reindexer/reindex_worker/src/main/scala/uk/ac/wellcome/platform/reindex/reindex_worker/models/ReindexSource.scala

```diff
@@ -0,0 +1,24 @@
+package uk.ac.wellcome.platform.reindex.reindex_worker.models
+
+import io.circe.Decoder
+
+import scala.util.{Failure, Success}
+
+sealed trait ReindexSource
+
+object ReindexSource {
+  case object Calm extends ReindexSource
+  case object Mets extends ReindexSource
+  case object Miro extends ReindexSource
+  case object MiroInventory extends ReindexSource
```

Comment:
what's the difference between miro and miro inventory?

Reply:
Miro = the metadata about Miro images, i.e. what gets fed to the transformer

Miro inventory = the metadata about where Miro images are stored. This only goes to the reporting cluster. I only discovered it when writing a test that we could parse the existing reindexer config.